Source/Scene/Globe.js

```diff
@@ -550,6 +569,36 @@ define([
         }
     };
 
+    /**
+     * @private
+     */
+    Globe.prototype.dirtyShaders = function() {
```

Comment:
We would generally name this a verb like `makeShadersDirty` and not make it on the `Globe` object; instead, it would be a stand-alone function at file level scope and then we pass in a `Globe` object.  This provides better encapsulation.